scrapers/il/events.py

```diff
@@ -112,14 +120,15 @@ def scrape(self):
                 no_scheduled_ct += 1
                 continue
 
-            tables = doc.xpath("//table[@width='550']")
+            tables = doc.xpath('//*[@id="pane-Week"]//table//tr')
             events = set()
             for table in tables:
-                meetings = table.xpath(".//a")
+                meetings = table.xpath(".//button")
                 for meeting in meetings:
-                    event, name = self.scrape_page(
-                        meeting.attrib["href"], chamber_names[chamber]
-                    )
+                    meeting_url = "https://beta.ilga.gov" + meeting.attrib[
```

Comment:
Please use a `BASE_URL` constant (or similar variable) in place of the literal string `https://beta.ilga.gov`. I expect this URL will change when they promote the beta site, and I'd like to be able to change one variable per scraper at that time (rather than having to go through code and find all the instances of this domain name string).

The `urls` array constant at the top of this file is OK